js-exercises/duplicate-letters/duplicateLetters.js

```diff
@@ -0,0 +1,37 @@
+function duplicateLetters(...args) {
+  if (typeof args[0] !== 'string') {
+    throw new Error('Sent parameter is not a string');
+  }
+  
+  try {
+    var arrOfString = args[0].split("");
+    let uniqueValofArr = new Array(...new Set(arrOfString));
+
+    // if all the values are unique - thats is no duplicate present
+    if (arrOfString.length == uniqueValofArr.length) {
+      return false;
+    }
+
+    let resultArr = [];
+
+    uniqueValofArr.forEach((ele) => {
+      let alpahetcount = 0;
+      resultArr.push(
+        arrOfString.reduce((acc, cur) => {
+          if (ele == cur || ele == acc) {
+            alpahetcount++;
+          }
+          return alpahetcount;
+        })
+      );
+    });
+
+    return resultArr.sort((a, b) => a - b).pop();
+  } catch (error) {
```

Comment:
You should not wrap your solution inside `try-catch` block every time. Normal usage of` try-catch` would be whenever you thing there could be unexpected error ( normally can happen  while doing network request  fetch, JSON.parse, JSON.stringify ).
But here you have just used Set, Array, reduce and nothing could go wrong inside JAVASCRIPT implementation of ECMAscript.

@ArfatSalman I am not 100% sure about it. Please provide your advice on it.